<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCaesarCipherMainDialog</class>
 <widget class="QDialog" name="QtCaesarCipherMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CaesarCipher</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/CaesarCipherArrowDown16x16.png</normaloff>:/images/CaesarCipherArrowDown16x16.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog { 
background-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0  #808080, stop: 1 #ffffff);
}
QWidget { font: 18px bold &quot;Courier New&quot;, courier, monospace }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QLabel" name="label_arrow_up">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ToolCaesarCipher.qrc">:/images/ToolCaesarCipherArrowUp34x34.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="button_deencrypt">
     <property name="text">
      <string>Deencrypt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="box_key">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>12345</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="button_encrypt">
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_arrow_down">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ToolCaesarCipher.qrc">:/images/ToolCaesarCipherArrowDown34x34.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLineEdit" name="edit_plaintext">
     <property name="text">
      <string>thisisplaintext</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_key">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QLineEdit" name="edit_encrypted_text"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ToolCaesarCipher.qrc"/>
 </resources>
 <connections/>
</ui>
